src/components/DragDrop/DragDrop.tsx

```diff
@@ -1,5 +1,6 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
+// @typescript-eslint/no-explicit-any
```

Comment:
What's the reason to changing DragDrop in this MR!? 🤔

Reply:
because the lint was failing, because the any prop was used